roles/ceph-grafana/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+##########
+# DOCKER #
+##########
+
+ceph_grafana_docker_extra_env:
```

Comment:
We probably need to move this into ceph-defaults like the other grafana variables (even if it also make sense to put this here) and run the `generate_group_vars_sample.sh` script

Reply:
Again, I followed what has been done for `ceph-osd` or `ceph-mon`; in those cases the variables are defined in the role's `defaults` directory, rather than `ceph-defaults`.

I can move this one to `ceph-defaults`, but that would make it a bit inconsistent; do you think it's the right solution nonetheless?